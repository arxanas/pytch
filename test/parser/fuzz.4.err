EXPECTED_PATTERN[1013] in test/parser/fuzz.4.pytch, line 1, character 4:
Error: I was expecting a pattern after 'let'.
   +-----------------------------------------------------+
   | test/parser/fuzz.4.pytch                            |
 1 | let let = print(1)                                  |
   |    ^ Error: I was expecting a pattern after 'let'.  |
   | ^~~ Note: This is the beginning of the let-binding. |
   +-----------------------------------------------------+
EXPECTED_PATTERN[1013] in test/parser/fuzz.4.pytch, line 1, character 9:
Error: I was expecting a pattern after 'let'.
   +---------------------------------------------------------+
   | test/parser/fuzz.4.pytch                                |
 1 | let let = print(1)                                      |
   |         ^ Error: I was expecting a pattern after 'let'. |
   |     ^~~ Note: This is the beginning of the let-binding. |
   +---------------------------------------------------------+
