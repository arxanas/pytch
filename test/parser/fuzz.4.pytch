let let = print(1)
