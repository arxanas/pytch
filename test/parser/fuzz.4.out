SyntaxTree
    LetExpr
        Token 'let'
        <missing>
        <missing>
        <missing>
        <missing>
        Leading ' let'
        Leading ' ='
        Leading ' print'
        Leading '('
        Leading '1'
        Leading ')\n'
        Leading ''
        Token DUMMY_IN ''
        <missing>
    Token EOF ''
